p2panda-sync/src/cbor.rs

```diff
@@ -54,17 +66,44 @@ where
     type Item = T;
     type Error = SyncError;
 
+    /// CBOR decoder method taking as an argument the bytes that has been read so far, and when it
+    /// is called, it will be in one of the following situations:
```

Comment:
```suggestion
    /// CBOR decoder method taking as an argument the bytes that have been read so far; when called,
    /// it will be in one of the following situations:
```